picard/coverart/processing/__init__.py

```diff
@@ -41,5 +54,25 @@ def run_image_metadata_filters(metadata):
     return True
 
 
-# def run_image_processors(data):
-#     pass
+def run_image_processors(data, coverartimage):
+    try:
+        start_time = time.time()
+        info = imageinfo.identify(data)
+        both, tags, file = get_cover_art_processors()
+        for processor in both:
+            data = processor.run(data, info, ProcessingTarget.BOTH)
+        tags_data = data
+        file_data = data
+        for processor in tags:
+            tags_data = processor.run(tags_data, info, ProcessingTarget.TAGS)
```

Comment:
`info` is not modified by processor, so it keeps initial values even though the processor changes those (for example, a processor changed dimensions, the next processor will get original dimensions, not the actual ones).

Reply:
You're right, sorry for missing that. I was thinking whether it makes more sense to use a `dataclass` then? Since they are mutable, each image processor does not have to import and create a new `ImageInfo` named tuple, but can instead modify whatever field it has changed. Does it make sense?

Reply:
Yes, it makes sense.